config.json

```diff
@@ -651,6 +651,19 @@
       "unlocked_by": "triangle",
       "uuid": "210bf628-b385-443b-8329-3483cc6e8d7e"
     },
+    {
+      "core": false,
+      "difficulty": 5,
```

Comment:
I still think this should be changed to 7 🙂

Reply:
Ok, but I think it is actually quite similar to two-bucket in difficulty in that all you have to do is a breadth first search.

Reply:
That's fair enough, they do look pretty similar. However, I think they're both quite a bit harder than most of the other difficulty 5 exercises, e.g. bob and pascals-triangle. Maybe the solution would be to also increase the difficulty of two-bucket? What do you think @jssander, @exercism/java? 🙂

Reply:
I'm not as familiar with the other exercises as you guys, so I'll leave this up to you.